galaxygpt/AiClient.cs

```diff
@@ -73,7 +73,7 @@ public partial class AiClient(
 
         string finalMessage = messages[^1].Content[0].Text;
         await ModerateText(finalMessage, moderationClient);
-        return (finalMessage, gptTokenizer.CountTokens(finalMessage));
+        return (finalMessage, gptTokenizer.CountTokens(messages.First().Content[0].Text), clientResult.Value.Usage.OutputTokenCount);
```

Comment:
this breaks tests (mocked clientResult doesnt have a value for OutputTokenCount most likely).

Reply:
its actually caused either by `gptTokenizer.CountTokens(messages.First().Content[0].Text)` or `clientResult.Value.Usage.OutputTokenCount`, being null, i dont know which one and i wont really be able to test this today

Reply:
its the latter. you dont have to change anything here, you'll need to change tests to provide a value for OutputTokenCount